### PR TITLE
pyktx (Python bindings for libktx)

### PR DESCRIPTION
**Pull request by ShukantPal**
#663

### CODE REVIEW

**Comment by ShukantPal** [comment]
Screenshot 2023-01-08 at 3 55 02 PM

Been pullin' my hair for a while and it turns out 64-bit Windows ships with 32-bit versions of Python. How convenient!

File: ci_scripts/build_win.ps1 [hunk 1]
```diff
@@ -20,7 +20,7 @@ function Set-Config-Variable {
     if ($res -eq $null) {
         $res = $DefaultValue
     }
-  } 
+  }
```
Comment (ShukantPal) [inline ci_scripts/build_win.ps1:23]:
```suggestion
  } 
```
(revert)

**Comment by MarkCallow** [comment]
I'm mystified by the VS2015 build issue. It shows as red but in the log it appears to have completed successfully. Don't worry about it. I shall be removing the VS2015 build from CI as soon as I've been able to release 4.1.0 and maybe sooner. Since I will not be able to merge this until after the release, there is no need to figure out what is wrong with VS2015.

How did you get the VS2017 build to find Python3? We've had issues  with that in another branch.

**Comment by ShukantPal** [comment]
I added unit tests to the Python bindings and binded them to the CTest suite 🙌. I believe this round of work (PR) is now complete from my side.

**Comment by ShukantPal** [comment]
Thanks for the update, will look into this.

**Pull request closed without merging**